### PR TITLE
[SYCLomatic][Doc] Update --out-root option.

### PR DESCRIPTION
**Pull request by tangjj11**
Signed-off-by: Tang, Jiajun jiajun.tang@intel.com

### CODE REVIEW

**Review by zhimingwang36 — APPROVED** [review]
LGTM

File: docs/_include_files/options_def.rst [hunk 1]
```diff
@@ -373,7 +373,8 @@ The directory path for root of generated files. A directory is created if
 it does not exist. Default: ``dpct_output``.
 
 The relative paths for the generated files are maintained, and the
-extension is changed as follows:
+extension is changed as follows by default, the behavior can be changed
```
Comment (krisdale) [inline docs/_include_files/options_def.rst]: Recommend the following edit: 

"The relative paths for the generated files are maintained. By default, file extensions are changed as follows:

*.cu → *.dp.cpp
....
*.h *.hpp *.hxx → extensions are kept the same


To limit file extension changes to ``.cu`` and ``.cuh`` files only, use the ``--change-cuda-files-extension-only`` option."

**Review by krisdale — CHANGES_REQUESTED** [review]
Approved with the recommended changes.

File: docs/_include_files/options_def.rst (hunk 1 above)
Comment (tangjj11) [reply docs/_include_files/options_def.rst]: fixed

**Review by krisdale — APPROVED** [review]
LGTM!

**Review by zhimingwang36 — APPROVED** [review]

**Pull request merged**